androidlibrary_lib/src/main/res/layout/progress_indicator_layout.xml

```diff
@@ -1,32 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
     <com.google.android.material.card.MaterialCardView
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="118dp"
```

Comment:
I am not sure how this will look with tablets. Remember the UI should work for 3 inch screens and 12 inch screens. Using DP solves some of the problems as it's pixel density but that is one factor of size of things. Having screenshot of how it looks on different size screens would help. NOTE: we do use dp for padding/margins because it's clear.

Reply:
@wbrunette thanks for the feedback, Working on that.